scripts/checksrc.pl

```diff
@@ -697,6 +698,11 @@ sub scanfile {
                       $line, length($1), $file, $ol, "no space before colon of switch label");
         }
 
+        if($prevl !~ /\?\z/ && $l =~ /^ +([A-Za-z][A-Za-z0-9]*):$/ && $1 ne 'default') {
```

Comment:
I think we're missing underscores:
```suggestion
        if($prevl !~ /\?\z/ && $l =~ /^ +([A-Za-z_][A-Za-z0-9_]*):$/ && $1 ne 'default') {
```